react_app/src/components/CTFCategory.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react'
+import { makeStyles } from '@material-ui/core/styles';
+
+import Typography from '@material-ui/core/Typography'
+import Paper from '@material-ui/core/Paper'
+import Grid from '@material-ui/core/Grid'
+import Divider from '@material-ui/core/Divider'
+
+import ChallengeCard from './ChallengeCard'
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    padding: "0.3em",
```

Comment:
You should change this to access the theme spacing rather than specifying literals, will allow you to change your mind later;
```
padding: theme.spacing(0.3)
```
That said, I think probably the reason you've put this extra padding on here is because you want gutters - in which case you can also use MUI to specify that more declaratively;

```
root: {
    ...theme.mixins.gutters
}
```
That wil set up a sensible gutter for you, then you can just change that to 0.3 globally if you disagree with their defaults :)

Reply:
Spacing works well, so I'll be using that solution